templates/user/dashboard/issues.tmpl

```diff
@@ -126,6 +126,10 @@
 										<span class="octicon octicon-checklist"></span> {{$tasksDone}} / {{$tasks}} <span class="progress-bar"><span class="progress" style="width:calc(100% * {{$tasksDone}} / {{$tasks}});"></span></span>
 									</span>
 								{{end}}
+								{{if ne .DeadlineUnix 0}}
+									<span class="octicon octicon-calendar"></span>
```

Comment:
```suggestion
									<span class="due_date poping up" data-content="{{$.i18n.Tr "repo.issues.due_date"}}" data-variation="tiny inverted" data-position="right center">
										<span class="octicon octicon-calendar"></span><span{{if .IsOverdue}} class="overdue"{{end}}>{{.DeadlineUnix.FormatShort}}</span>
```

Reply:
add add a 'padding-left: 5px;' to due_date so due_date and user dont stick together

Reply:
Sure, update the repo issues list to match as well